www/front_src/src/Navigation/Sidebar/Menu/MinCollaps.tsx

```diff
@@ -0,0 +1,283 @@
+import React, { useState } from 'react';
+
+import { equals } from 'ramda';
+import clsx from 'clsx';
+import { useAtom } from 'jotai';
+
+import Collapse from '@mui/material/Collapse';
+import List from '@mui/material/List';
+import makeStyles from '@mui/styles/makeStyles';
+import ListSubheader from '@mui/material/ListSubheader';
+
+import { Page } from '../../models';
+import { itemSelectedAtom, propsItemSelected } from '../sideBarAtoms';
+
+import ListButton from './ListButton';
+
+interface CollapsProps {
+  currentTop?: number;
+  currentWidth: number;
+  data?: Array<Page>;
+  isCollapsed: boolean;
+  isSubHeader?: boolean;
+  level: number;
+  onClick: (item: Page) => void;
+}
+
+interface StyleProps {
+  currentTop?: number;
+  currentWidth: number;
+}
+
+const collapsWidth = 170;
+
+const useStyles = makeStyles((theme) => ({
+  activated: {
+    '& .MuiListItemText-root': {
+      '& .MuiTypography-root': {
+        color: theme.palette.background.paper,
+      },
+    },
+    '& .MuiSvgIcon-root': {
+      color: theme.palette.background.paper,
+    },
+    '&:hover': {
+      backgroundColor: theme.palette.primary.main,
+    },
+
+    backgroundColor: theme.palette.primary.main,
+  },
+  label: {
+    fontWeight: 'bold',
+  },
+  root: {
+    '& .MuiListItemIcon-root': {
+      minWidth: theme.spacing(2.25),
+      padding: theme.spacing(0, 0.5, 0, 1),
+    },
+    '& .MuiTypography-root': {
+      color: theme.palette.text.primary,
+      fontSize: theme.typography.caption,
+    },
+    border: `solid ${theme.palette.divider} 0.1px`,
+  },
+  scroll: {
+    '&::-webkit-scrollbar': {
+      width: theme.spacing(1.2),
+    },
+    '&::-webkit-scrollbar-thumb': {
+      backgroundColor: theme.palette.action.disabled,
+    },
+    '&::-webkit-scrollbar-track': {
+      border: `solid ${theme.palette.action.hover} 0.5px`,
+    },
+    maxHeight: theme.spacing(44),
+    overflowX: 'hidden',
+  },
+  subHeader: {
+    color: theme.palette.text.secondary,
+    fontSize: theme.typography.body2.fontSize,
+    fontWeight: 'bold',
+    lineHeight: theme.spacing(3),
+    textAlign: 'center',
+  },
+  toggled: {
+    backgroundColor: theme.palette.background.default,
+    left: ({ currentWidth }: StyleProps): string => theme.spacing(currentWidth),
+    minWidth: collapsWidth,
+    position: 'fixed',
+    top: ({ currentTop }: StyleProps): number | undefined => currentTop,
+    zIndex: theme.zIndex.mobileStepper,
+  },
+}));
+
+const MinCollaps = ({
```

Comment:
MonCollaps doesn't sound clear to me, even though we're using the Collapse component under the hood. What about CollapsableItems?